### DISCUSSION TITLE
There is an already successful Forge fix

### DISCUSSION THREAD

**Discussion by Unloaded123456** [Abstrat-Technologies/rustedanvil#34] [General]
I recently found a project that successfully fixed Minecraft Forge 1.4.7, I tested and its first a few commits work but there is a problem in the last commit that I could not understand. Instead of attempting to fix the completely broken versions of Forge, we could modify that project to be compatible with 1.6.4 which seems to be a lot easier. The project: [https://github.com/Su5eD/MinecraftForge](https://github.com/Su5eD/MinecraftForge)

**Comment by arek-kwapis** [comment]
This is one of the early repositories I looked at when trying to find information for a fix, before doing my own. Unfortunately, it will not be as easy as you think, hence why I'm going to disregard it. In no particular order:

1. There are changes between 1.4.7 and 1.6.4 which means some if not all the code would need to be redone to get it to work
2. It uses gradle. I've already explained multiple times why I'm against using it, and will not develop for it
3. I've invested so much time already into fixing 1.6.4 the way we're doing it now that, quite honestly, I'd just call it quits before having to start again. Especially with how far along it is
